ISISComputingGroup/ibex_gui
# Ticket369: Add keyboard shortcuts for perspectives

### Description of work

Adds shortcuts for all visible perspectives on the function keys.

### To test

See https://github.com/ISISComputingGroup/IBEX/issues/369

Confirm that all visible perspectives can be switched to using the function keys. Ensure this still works when your client has non-default perspectives available.

Added manual test 27-1 & 27-2

---

#### Code Review

_hard to unit test due to use of extension points_

### Functional Tests


### Final Steps
